index.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
+import Book from './modules/bookModule.js';
+import { homeClick, addClick, contactClick } from './modules/eventHandlerModule.js';
+import { DateTime } from './modules/luxon/build/es6/luxon.js';
+
+window.getCurrentDateTime = function () {
+  return DateTime.local();
+};
+
+window.formatDateTime = function (dateTime, format) {
+  return dateTime.toFormat(format);
+};
+
+const updateDateTime = function () {
+  const currentDateTime = window.getCurrentDateTime();
+  const formattedDateTime = window.formatDateTime(currentDateTime, 'yyyy-MM-dd HH:mm:ss');
```

Comment:
- [ ] **Optional:** Some of these functions can also be converted to arrow functions as you've done in the `eventHandler` file.